slick.dataview.js

```diff
@@ -1053,6 +1053,7 @@
       }
     }
 
+    var selectedRowIds = null;
```

Comment:
you should move this variable declaration to the top (around line 74)

Reply:
sure, will do!